invenio_administration/menu/menu.py

```diff
@@ -62,6 +65,20 @@ def register_menu_entries(self, flask_menu_instance, menu_key="admin_navigation"
                     icon=icon,
                 )
 
+    def register_admin_entry(self, current_menu, endpoint):
+        """Register administration entry as the last one."""
+        current_menu.submenu("login.administration").register(
```

Comment:
Question: IMHO not sure `login` is the most intuitive name, maybe `navbar-menu` would be more explicit? WDYT?